ggml-metal.m

```diff
@@ -911,7 +925,9 @@ void ggml_metal_graph_compute(
                                             nth1 = 1;
                                             if (ne11 * ne12 < 4) {
                                                 [encoder setComputePipelineState:ctx->pipeline_mul_mat_f16_f32_1row];
-                                            } else if (ne00 >= 128 && ne01 >= 8 && ne00%4 == 0) {
+                                            //} else if (ne00 >= 128 && ne01 >= 8 && ne00%4 == 0) {
```

Comment:
So, if you are sure that it is no longer needed, then remove it altogether. If you are not sure, then this change does not make sense.

Reply:
Will do some more tests - for now keeping the original implementation.

With this PR, `nrows` is always `1` because `ne11` is `1`, so I think it would either need to be updated or probably it is not needed anymore.